redis/files/redis-2.4.conf.jinja

```diff
@@ -106,423 +41,86 @@ save 300 10
 save 60 10000
```

Comment:
To be honest i just copied this lines from redis-2.6|8.conf.jinja, as these files seem to me more well-kept than the 2.4-version. If we don't want this behavior, I think whe should change it in all config-templates.